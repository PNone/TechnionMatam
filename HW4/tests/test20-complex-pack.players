Daniel Warrior Responsible
Regev Magician Responsible