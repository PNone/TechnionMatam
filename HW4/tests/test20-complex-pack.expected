Game is about to start!
These are the players:

1. Daniel, Warrior with Responsible character (level 1, force 5)
2. Regev, Magician with Responsible character (level 1, force 5)

----------------------------------------

New round starts!

Turn: 1
Player: Daniel, Warrior with Responsible character (level 1, force 5)
Event: Snail (power 5, loot 2, damage 10)
Daniel won the encounter, gained 2 coins and leveled up!

Turn: 2
Player: Regev, Magician with Responsible character (level 1, force 5)
Event: Pack of 4 members (power 115, loot 329, damage 27148)
Regev lost the encounter and took 27148 damage!

Round has ended!
Leader board is as follows:
Ranking    Player Name     Level     Force HP    Coins
1          Daniel          2         5     140   12
2          Regev           1         5     0     10

----------------------------------------

New round starts!

Turn: 3
Player: Daniel, Warrior with Responsible character (level 2, force 5)
Event: Balrog (power 15, loot 100, damage 9001)
Daniel lost the encounter and took 9001 damage!

Round has ended!
Leader board is as follows:
Ranking    Player Name     Level     Force HP    Coins
1          Daniel          2         5     0     12
2          Regev           1         5     0     10

----------------------------------------

Game is over!
There are no winners in this game.
Better luck next time!
